nx/lib/nx.ex

```diff
@@ -8451,6 +8451,53 @@ defmodule Nx do
     )
   end
 
+  ## Sigils
+
+  if Version.match?(System.version(), ">= 1.13.0-dev") do
```

Comment:
We don't need this check. We can always define the sigil but we will note in the docs the type modifiers require 1.13.